demos/action_recognition_demo/python/README.md

```diff
@@ -130,6 +130,13 @@ python3 action_recognition_demo.py \
     -lb <omz_idr>/demos/action_recognition_demo/python/driver_actions.txt
 ```
 
+When single image applied as an input, the demo will process and render it quickly, then exit. In this particular case, recommendation is to also apply `loop` option, which will enforce looping over processing the single image, so processed results will be continuously visualized on screen.
+The demo allow saving of processed results to a Motion JPEG AVI file or separate JPEG or PNG files when `-o` option is used. To save processed results in AVI file, the name of output file with `avi` extension should be specified with `-o` option, for example: `-o output.avi`. To save processed results as an images, the template name of output image file with `jpg` or `png` extension should be specified with `-o` option, as shown on example: `-o output_%03d.jpg`. The actual file names will be constructed from template at runtime by replacing regular expression `%03d` with frame number, resulting in storing files with names like following: `output_000.jpg`, `output_001.jpg`, and so on.
+Amount of data stored in output file or files could be limited with `limit` option, to avoid disk space overrun in case of continouus input stream, like camera. Default value is 1000, it can be changed by applying `-limit N` option, where N is number of frames to store.
```

Comment:
```suggestion
Amount of data stored in output file or files could be limited with `limit` option, to avoid disk space overrun in case of continuous input stream, like camera. Default value is 1000, it can be changed by applying `-limit N` option, where N is number of frames to store.
```